src/chapters/06/main.tex

```diff
@@ -113,27 +130,34 @@ \section{Theory}\label{sec:theory}
 \label{fig:taxi-firm-game}
 \end{figure}
 
+A strategy corresponds to an way of choosing actions, this is represented by a
+probability vector. For the \(i\)th player, this vector \(v\) would be of size
+\(|A_i|\) (the size of the action space) and \(v_i\) corresponds to the
+probability of choosing the \(i\)th action.
+
 Both taxis always choosing to use 2 taxis (the second row/column) would
 correspond to the strategy: \((0, 1, 0)\).
 If the both companies use this strategy and the row player (who controls the
-rows) wants to
-improve their outcome it's evident by inspecting the second column that the
-highest number is \(19 / 20\): thus the row player has no reason to change what
-they are doing.
+rows) wants to improve their outcome it's evident by inspecting the second
+column that the highest number is \(19 / 20\): thus the row player has no reason
+to change what they are doing.
 
-This is in fact called a Nash equilibrium: when both players are playing a
+This is called a Nash equilibrium: when both players are playing a
 strategy that is the best response against the other.
 
-Whilst a Nash equilibria is not necessarily a set of strategies that players
+Whilst a Nash equilibrium is not necessarily a set of strategies that players
 will converge towards, once they are there they have no reason to move away from
-it. It is the particular concept we will use to understand the emergent
-behaviour in our city.
+it. It is the particular concept that will be used to understand the emergent
+behaviour in the city.
+
+% TODO I don't thnk this last paragraph is needed, or makes sense in this context. We don't yet know what you mean by 'converge', we don't have any concept of repeating a game and learning from the outcomes etc.
+
+% TODO Having said that, I don't think there is enough theory here, especially compared to other chapters, e.g. the LP chapter or the DES chapter. We do not know how to find a Nash equilibrium. We don't know if a Nash equilibrium necessarily exists. Can something else be added?
```

Comment:
How about something like::

```suggestion
An important fact is that a Nash equilibrium is guaranteed to exist. 
This was actually the theoretic result for which John Nash received a noble prize.
% TODO Add footnote reference

There are various algorithms that can be used for finding Nash equilibria, they involve
a search through the pairs of spaces of possible strategies until pairs of best 
responses are found. Mathematical insight allows this do be done somewhat efficiently
using algorithms that can be thought of as modifications of the algorithms used in Linear Programming.
% TODO Point at LP chapter and add footnote references.
```

Reply:
Is it worth mentioning more about mixed strategy equilibria in the theory?

In some sense it's self evident but perhaps this will free up/remove some of the discussion in the code.

Reply:
I've decided to leave as is, happy to come back around to this if you'd like to @geraintpalmer